tests/testconf-example.json

```diff
@@ -0,0 +1,11 @@
+{
+    "topic": "<test-topic> or $TOPIC env",
```

Comment:
it isn't clear here that any environment variable will work, maybe add that as a comment above ("__doc": "Any value starting with $ will be treated like a environment variable and its value will be automatically retrieved"

Reply:
How about adding it to tests/README.md instead

Reply:
That should absolute be stated in the test README, but it is also good with an example in the example, .e.g, `"topic": "$MY_TOPIC_ENV"`, notice how the naming makes it more clear that any env can be used